src/components/Auth/ForgotPassword/ForgotPassword.react.js

```diff
@@ -1,234 +1,132 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
-/* Material-UI*/
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import Paper from 'material-ui/Paper';
 import TextField from 'material-ui/TextField';
 import RaisedButton from 'material-ui/RaisedButton';
 import Dialog from 'material-ui/Dialog';
 import Close from 'material-ui/svg-icons/navigation/close';
-
-/* CSS*/
-import './ForgotPassword.css';
-
-/* Utils*/
-import $ from 'jquery';
 import UserPreferencesStore from '../../../stores/UserPreferencesStore';
```

Comment:
Is it required now?

Reply:
User preference is being used to get themes colors in render function for now. If it had not been used, Eslint would have pointed it out.